tests/components/controlBar-test.js

```diff
@@ -272,20 +252,20 @@ describe('ControlBar', function() {
       />
     );
 
-    var volumeButton = TestUtils.findRenderedDOMComponentWithClass(DOM, 'oo-mute-unmute').firstChild;
-    TestUtils.Simulate.click(volumeButton);
+    var volumeButton = wrapper.find('.oo-mute-unmute').hostNodes();
+    volumeButton.simulate('click');
     expect(muteClicked).toBe(true);
-    var volumeBars = TestUtils.scryRenderedDOMComponentsWithClass(DOM, 'oo-volume-bar');
+    var volumeBars = wrapper.find('.oo-volume-bar').hostNodes();
     // JEST doesn't support dataset at the time of writing
```

Comment:
I think this comment might be obsolete now

Reply:
Removed